pkg/chartutil/save.go

```diff
@@ -100,8 +128,64 @@ func SaveDir(c *chart.Chart, dest string) error {
 //
 // This returns the absolute path to the chart archive file.
 func Save(c *chart.Chart, outDir string) (string, error) {
+	tgzWriter, err := createTarGzipWriter(c, outDir)
```

Comment:
Can't we replace all of this in `Save` with a call to `SaveWithOpts(c, outDir)`? The code in `Save` and `SaveWithOpts` are nearly identical. We try to prevent adhering to the WET (Write Everything Twice) principle wherever possible.

Reply:
@bacongobbler Sure thing. I'll revise here shortly

Reply:
@bacongobbler this has been resolved in PR #8216 (on which this PR is based). This PR has been rebased to include the changes there